Doc/library/string.rst

```diff
@@ -588,6 +589,14 @@ The available presentation types for :class:`float` and
    |         | as altered by the other format modifiers.                |
    +---------+----------------------------------------------------------+
 
+The available presentation types for :class:`complex` are same as for
+:class:`float`.  Both real and imaginary components of a complex number are
+formatted as floating-point numbers, accordingly specified presentaion type.
+They separated by mandatory sign of the imaginary part, terminated by ``j``
+suffix.  If presentation type is missing --- result match the output of
+:func:`str` (numbers also surrounded by round parentheses, if real part is
+non-zero), altered by other format modifiers.
```

Comment:
```suggestion
The available presentation types for :class:`complex` are the same as for
:class:`float`. The real and imaginary components of a complex number are
formatted as floating-point numbers, according to the presentation type.
They are separated by a mandatory sign of the imaginary part suffixed by ``j``
(e.g., ``1.3 - 3.7j``).  If the presentation type is not specified, the result
matches the output of :func:`str` (numbers are surrounded by round parentheses,
if the real part is non-zero), possibly altered by other format modifiers.
```

Reply:
Does make sense for me, but I would prefer to avoid examples, as for other format types.

Reply:
I've updated the suggestion.